edge/pkg/edged/fake/typed/core/v1/fake_core_client.go

```diff
@@ -27,3 +27,11 @@ func (c *FakeCoreV1) PersistentVolumeClaims(namespace string) corev1.PersistentV
 func (c *FakeCoreV1) ConfigMaps(namespace string) corev1.ConfigMapInterface {
 	return &FakeConfigMap{fakecorev1.FakeConfigMaps{Fake: &c.FakeCoreV1}, namespace, c.MetaClient}
 }
+
+func (c *FakeCoreV1) Secrets(namespace string) corev1.SecretInterface {
+	return &SecretBridge{fakecorev1.FakeSecrets{Fake: &c.FakeCoreV1}, namespace, c.MetaClient}
+}
+
+func (c *FakeCoreV1) ServiceAccounts(namespace string) corev1.ServiceAccountInterface {
+	return &ServiceAccountsBridge{fakecorev1.FakeServiceAccounts{Fake: &c.FakeCoreV1}, namespace, c.MetaClient}
+}
```

Comment:
Add an empty line, for other types like ConfigMap, we will name it to Bridge in other prs？

Reply:
> Add an empty line
> 
> For other types like ConfigMap, we will name it to Bridge in other prs？

ConfigMapBridge rename in pr #4112